spec/models/page_spec.rb

```diff
@@ -320,6 +321,26 @@
       new_page = create :page
       expect(page.follow_up_plan).to eq 'with_liquid'
     end
+
+    context 'a page with a layout that has a default follow-up layout' do
+      it 'uses the correct layout for the follow-up page' do
+        new_page = PageBuilder.create(page_params)
```

Comment:
I don't think these specs really belong in the page_spec at all - they're really specs for PageBuilder.